templates/common/packages/hooks/localization/useLocalizedContent.test.ts

```diff
@@ -0,0 +1,61 @@
+import useLocalizedContent from './useLocalizedContent';
+import { renderHook } from '@testing-library/react-hooks';
+import * as helper from '../helper';
+
+class LocalStorageMock {
```

Comment:
This might be needed again and again, shall we not put this in global Mock?

Reply:
I think yes, we should. I will raise a seperat PR for this. Thanks!